### ISSUE TITLE
Reports page does not display properly on phone screens

### ISSUE THREAD

**Issue by jsweet-dev** [jsweet-dev/BTCMiningMonitor#8]
Columns are off the screen, some data does not display. Needs updates to make it look better and display the most relevant data in the screen space available. Starts getting mangled at less widths less than 700px and is really bad at widths less than 550px. The datepicker also seems to be fixed width so on a small screen the calendar portion of the picker only shows part of the left month.

Perhaps at smaller widths we can exclude the Outage End column (leaving name, start, length and the Details button).

Maybe also explore alternative UI element for the details button that doesn't require text and is smaller in width. 

The totals row text also gets cut off once the text starts wrapping.

**Comment by jsweet-dev** [comment]
I have started work on this in the reportsTableDesign branch. I plan to break this into at least three updates, one for the outage table and related design of the ReportPage component, one for addressing the datepicker being too wide for small screens, and a third for updating the OutageDetail component to be more responsive as well. I'm focusing reportsTableDesign on the ReportPage component first and then will create new branches for the other pieces of the work.

**Comment by jsweet-dev** [comment]
I've merged in reportsTableDesign to address the first piece of this work. I will next work on the datepicker in the SearchBar component in the responsiveDatepicker branch.

**Comment by jsweet-dev** [comment]
I've merged in another branch to fix the DateRangePicker width being too wide for small screens. NOTE: While working on this I discovered that react-date-range doesn't have touchscreen support at this time, so mobile device usage takes more taps to select a date range. 

Next I will look at updating the OutageDetail component. The first issue being the table at the top of the page is too wide and is not responsive to scaling down.
